app/views/pages/home.html.erb

```diff
@@ -64,18 +64,18 @@
 
 </div>
 <footer class="footer_wide">
-  <a class="logo footer-logo" href="/" title="Home">
-  <%= image_tag "AI_pic.jpg" %>
+  <%# <a class="logo footer-logo" href="/" title="Home"> %>
```

Comment:
Simply remove the commented out lines if they are no longer used

Reply:
pushed the changes